src/map/utils/mobutils.cpp

```diff
@@ -427,22 +427,22 @@ namespace mobutils
         }
         else
         {
-            sSTR = 0;
-            sDEX = 0;
-            sAGI = 0;
-            sINT = 0;
-            sMND = 0;
-            sCHR = 0;
-            sVIT = 0;
-        }
-
-        PMob->stats.STR = fSTR + mSTR + sSTR;
-        PMob->stats.DEX = fDEX + mDEX + sDEX;
-        PMob->stats.VIT = fVIT + mVIT + sVIT;
-        PMob->stats.AGI = fAGI + mAGI + sAGI;
-        PMob->stats.INT = fINT + mINT + sINT;
-        PMob->stats.MND = fMND + mMND + sMND;
-        PMob->stats.CHR = fCHR + mCHR + sCHR;
+            sSTR /= 4;
+            sDEX /= 4;
+            sAGI /= 4;
+            sINT /= 4;
+            sMND /= 4;
+            sCHR /= 4;
+            sVIT /= 4;
+        }
+
+        PMob->stats.STR = floor(fSTR) + floor(mSTR) + floor(sSTR);
```

Comment:
All of these look like they're declared as uint values.  There's no reason to floor